frontend/Dockerfile.prod

```diff
@@ -37,34 +37,32 @@ ARG APP_ENV
 ENV NEXT_PUBLIC_APP_ENV $APP_ENV
 ENV BABEL_ENV $APP_ENV
 
-RUN yarn build \
-    && yarn install --production --ignore-scripts --prefer-offline
+RUN yarn build 
```

Comment:
No longer necessary to install as node_modules as the standalone build output will contain everything needed to run the server.